lib/mix/tasks/templates/job.exs.eex

```diff
@@ -0,0 +1,5 @@
+use Kitto.Job.DSL
+
+job :<%= name %>, every: {5, :seconds} do
+  broadcast! :<%= name %>, %{}
```

Comment:
I agree. I think my original thought was to be more verbose, but there's no real need to be.